Formula/apibuilder-cli.rb

```diff
@@ -0,0 +1,41 @@
+class ApibuilderCli < Formula
+  desc "Command-line interface to apibuilder"
+  homepage "https://www.apibuilder.io"
+  url "https://github.com/apicollective/apibuilder-cli/archive/0.1.4.tar.gz"
+  sha256 "8267dce31b951987ae2395be23180fe330ea2056c2f765ecb066736713378294"
+
+  def install
+    inreplace "bin/apibuilder", "File.join(File.dirname(__FILE__), 'apibuilder-cli-config')", "'#{prefix}/scripts/apibuilder-cli.rb'"
```

Comment:
We'll need upstream to handle this case without us needing to do string replacements in the formula.

Reply:
fixed with updated commit and https://github.com/apicollective/apibuilder-cli/pull/22